### PR TITLE
Add unhide button

### PR DESCRIPTION
**Pull request by mariuscoto**

### CODE REVIEW

**Comment by mariuscoto** [comment]
@marianitadn please have a look

File: views/challenge.jade [hunk 1]
```diff
@@ -107,10 +107,11 @@ block content
 
                 - if (user && user.admin)
                   - if (p.hide)
-                    input.btn.btn-success(type='submit', value='Hidden', disabled)
+                    a(href='/challenges/' + challenge.link + '/unhide_commit?id=' + p._id)
+                      input.btn.btn-warning(type='submit', value='Hidden')
                   - else
                     a(href='/challenges/' + challenge.link + '/hide_commit?id=' + p._id)
-                      input.btn.btn-success(type='submit', value='Hide')
+                      input.btn.btn-success(type='submit', value='Visible')
```
Comment (marianitadn) [inline views/challenge.jade]: "Display" like the event. :)

File: app.js [hunk 2]
```diff
@@ -98,6 +98,7 @@ app.get('/challenges/:ch/pulls', challenge.one);
 app.get('/challenges/:ch/join', ensureAuth, challenge.join);
 app.get('/challenges/:ch/refresh', ensureAuth, challenge.refresh);
 app.get('/challenges/:ch/hide_commit', ensureAuth, challenge.hide_commit);
+app.get('/challenges/:ch/unhide_commit', ensureAuth, challenge.unhide_commit);
```
Comment (marianitadn) [inline app.js]: I would go for "display_commit", it's the opposite action of "hide" :)

File: routes/challenge.js [hunk 3]
```diff
@@ -241,6 +241,27 @@ exports.hide_commit = function(req, res) {
...
+
+  function gotChallenge(err, ch) {
+    // Check if user is admin
+    if (ch.admins.indexOf(req.session.auth.github.user.login) < 0)
+      return res.redirect('/challenges/' + req.params.ch)
+
+    var conditions = {'pulls._id': new mongoose.Types.ObjectId(req.query.id)}
+    var update = {$set: {'pulls.$.hide': false}}
+    Challenges.update(conditions, update, function (err, num) {
+      console.log("* Unhide commit " + req.query.id)
```
Comment (marianitadn) [inline routes/challenge.js]: What does `console.log`? It's like a logger?

**Pull request merged**